pkg/expression/column.go

```diff
@@ -88,6 +88,11 @@ func (col *CorrelatedColumn) Traverse(action TraverseAction) Expression {
 	return action.Transform(col)
 }
 
+// VecEvalVectorFloat32 evaluates this expression in a vectorized manner.
+func (col *CorrelatedColumn) VecEvalVectorFloat32(ctx EvalContext, input *chunk.Chunk, result *chunk.Column) error {
+	return genVecFromConstExpr(ctx, col, types.ETVectorFloat32, input, result)
+}
+
```

Comment:
Put them before `func (col *CorrelatedColumn) Traverse(action TraverseAction) Expression {`. Group all `VecEvalXxx` together just like previous PRs.

Reply:
Get. I've moved the code position.